GiniVision/Classes/Core/GiniScreenAPICoordinator.swift

```diff
@@ -42,6 +42,11 @@ final class GiniScreenAPICoordinator: NSObject, Coordinator {
     fileprivate(set) var documentRequests: [DocumentRequest] = []
     fileprivate let multiPageTransition = MultipageReviewTransitionAnimator()
     weak var visionDelegate: GiniVisionDelegate?
+    
+    // When there was an error uploading a document or analyzing it and the analysis screen is not initialized yet,
```

Comment:
If have too keep the tense between the two sub-sentenses. If one has "was" the other needs to also be in past tense. For instance:

> When there was an error uploading a document or analyzing it and the analysis screen hadn't been initialized yet

Also applies to the other comment in the component API, which is a copy of this one.